src/ontology/mondo-edit.obo

```diff
@@ -538405,6 +538416,15 @@ is_a: MONDO:0005583 ! non-human animal disease
 intersection_of: MONDO:0005583 ! non-human animal disease
 intersection_of: MONDO:0700097 MONDO:0021084 ! cross-species analog vision disorder
 
+[Term]
+id: MONDO:1030001
+name: epilepsy, juvenile absence, susceptibility to
+xref: OMIMPS:607631 {source="MONDO:equivalentTo"}
+is_a: MONDO:0020573 ! inherited disease susceptibility
+relationship: has_characteristic MONDO:0021152 ! inherited
```

Comment:
you can add the logical definition
'inherited disease susceptibility' and ('predisposes towards' some 'juvenile absence epilepsy')

Reply:
fixed